<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjDeclEntry</class>
 <widget class="QDialog" name="AdjDeclEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Russian Adjectival Declension Entry</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>780</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Russian Adjectival Declension</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="nomEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nomLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nominative</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="genLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Genitive</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="genEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="insLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Instrumental</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="insEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="accLbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Accusative</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="accEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="datLbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dative</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="datEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="prpLbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Prepositional</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LineEditRU" name="prpEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectBttn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>780</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>770</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="anim">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="inan">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Inanimate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fem">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>130</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Feminine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nuet">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nueter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pluranim">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Plural Animate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="plurinan">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Plural Inanimate</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="LineEditRU" name="stemEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stemLbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stem</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="engLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="LineEditEN" name="engEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="declBttn">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>530</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Declensions</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditRU</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineeditru</header>
  </customwidget>
  <customwidget>
   <class>LineEditEN</class>
   <extends>QLineEdit</extends>
   <header>machinetrans.lineediten</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>anim</tabstop>
  <tabstop>inan</tabstop>
  <tabstop>fem</tabstop>
  <tabstop>nuet</tabstop>
  <tabstop>pluranim</tabstop>
  <tabstop>plurinan</tabstop>
  <tabstop>engEdit</tabstop>
  <tabstop>stemEdit</tabstop>
  <tabstop>nomEdit</tabstop>
  <tabstop>accEdit</tabstop>
  <tabstop>genEdit</tabstop>
  <tabstop>datEdit</tabstop>
  <tabstop>insEdit</tabstop>
  <tabstop>prpEdit</tabstop>
  <tabstop>rejectBttn</tabstop>
  <tabstop>declBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
